Makefile

```diff
@@ -1,6 +1,6 @@
 buildroot = /
 docdir = /usr/share/doc/packages
-python_version = 3
+python_version = 3.11
```

Comment:
Does hardcoding 3.11 here make sense? Wouldn't that break when running under python 3.12?

Reply:
Ah good catch, no doesn't make sense and was only a temporary change while I was testing the package. Fixed it.